x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/elasticsearch/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import React, { Fragment } from 'react';
+import {
+  EuiSpacer,
+  EuiCodeBlock,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiCallOut,
+  EuiText
+} from '@elastic/eui';
+import { Monospace } from '../components/monospace';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { statusTitle } from './common_elasticsearch_instructions';
+
+export function getElasticsearchInstructionsForEnablingMetricbeat(product, _meta, {
+  esMonitoringUrl,
+  hasCheckedStatus,
+  checkingMigrationStatus,
+  checkForMigrationStatus,
+  autoCheckIntervalInMs
+}) {
+  const installMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.installMetricbeatTitle', {
+      defaultMessage: 'Install Metricbeat on the same server as Elasticsearch'
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.installMetricbeatDescription"
+            defaultMessage="Follow {link}."
+            values={{
+              link: (
+                <EuiLink href="https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-installation.html" target="_blank">
```

Comment:
Is it possible to link to the appropriate version instead of "current"?  Otherwise it could point to an inappropriate version of the documentation.

Reply:
I think this is a good idea, but I feel it's a bit complicated given our links in documentation. Currently, `current` is `8.0.0` and `7.x` represents every `7.x` release _after_ `7.2` but this changes after each release which means I'd have to update this code after every release too.

Thoughts @ycombinator?

Reply:
You don't need to update the code in every release. There's a utility in Kibana to give you the appropriate documentation links for the version of Kibana that's running. Here's some examples of it:

* Security doc links: https://github.com/elastic/kibana/blob/master/x-pack/plugins/security/public/documentation_links.js
* Watcher UI doc links: https://github.com/elastic/kibana/blob/master/x-pack/plugins/watcher/public/lib/documentation_links/documentation_links.js

There are others like this too so you might want to take a look around and go with something that makes most sense for Stack Monitoring.

Reply:
Thanks @ycombinator!